lib/matestack/ui/vue_js/components/async.js

```diff
@@ -39,6 +39,11 @@ const componentDef = {
     hide: function(){
       this.showing = false
       this.event.data = {}
+      if(this.props["hide_after"] != undefined) {
```

Comment:
I don't think you'll have to touch this component though, the `hide` method is already called in a timeout in line 35 if `hide_after` is defined

Reply:
Yes this component does not need to be touched. The hide things in the `async` component should actually be deleted at some point...